File: HackerHank/PickingNumbers/PickingNumbersSolution.cpp
#include <vector>
using namespace std;
int pickingNumbers(vector<int> a) 
{
    vector<int> freq;
    for(int i=0;i<101;i++)
        freq.push_back(0);
    for(int i=0;i<a.size();i++)
        freq[a[i]]++;
    int max=0;
    //sort(freq.begin(),freq.end());
    for (int i = 1; i <= 100; i++) 
    {
            int count = freq[i] + freq[i - 1];
            if (count > max) {
                max = count;
            }
    }
    return max;
}

File: HackerHank/PickingNumbers/PickingNumbersSolution.py
def pickingNumbers(a):
    freq = [0]*101
    for i in range(0,len(a)):
        freq[a[i]]+=1
    maximum = 0
    for i in range(1,101):
        count = freq[i] + freq[i - 1];
        if (count > maximum):
              maximum = count;
    return maximum
    # Write your code here